commit ce828c400054af96f61b41daa56bd08c195ac749
Author: Fabrizio Fabbri <ffabbri4@bloomberg.net>
Date:   Wed Mar 2 12:43:36 2016 -0500

    Win64 and Visual Studio compiler
    
    * Fix compilation error.
    * Fix feature discover on cl compiler.

diff --git a/src/cmp/cmpmain.lsp b/src/cmp/cmpmain.lsp
index 2eb4008..0b8bbfd 100755
--- a/src/cmp/cmpmain.lsp
+++ b/src/cmp/cmpmain.lsp
@@ -234,6 +234,7 @@ the environment variable TMPDIR to a different value." template))
 extern \"C\"
 #endif
 
+ECL_DLLEXPORT
 void ~A(cl_object cblock)
 {
         /*
